index.src.html

```diff
@@ -865,22 +906,14 @@ <h3 id="determine-requests-referrer" dfn export>
         <dd>
           <ol>
             <li>
-              If <var>request</var> is a <a>same-origin request</a>, then
-              return <var>referrerURL</var>.
+              If the  <a for=url>origin</a> of |referrerURL| and the <a for=url>origin</a> of
+              <var>request</var>'s <a for=request>current url</a> are <a lt="same origin">the
+              same</a>, then return |referrerURL|.
```

Comment:
optional nit: could add a note here saying that this determines whether the request is same-origin-referrer (to make the definition of same-origin-referrer completely concrete)

Reply:
Thanks, done.